### PR TITLE
Updates played minutes timer and login screen

### PR DESCRIPTION
**Pull request by runelaenen**
The timer only polls the server when the process has quit instead of every 5 minutes.
Also added a link to OpenRCT.net on the login window.

### CODE REVIEW

**Comment by runelaenen** [comment]
I also wrote a small update script (just notify the user if an update was available) but reverted it since I think it should be in a different pull request. It also wasn't perfect

**Comment by Krutonium** [comment]
Ready for merge?

File: Launcher/Launcher/Launcher.Designer.vb [hunk 1]
```diff
@@ -39,19 +39,19 @@ Partial Class frmLauncher
         Me.cmdLaunchGame.Enabled = False
         Me.cmdLaunchGame.Location = New System.Drawing.Point(12, 158)
         Me.cmdLaunchGame.Name = "cmdLaunchGame"
-        Me.cmdLaunchGame.Size = New System.Drawing.Size(110, 37)
-        Me.cmdLaunchGame.TabIndex = 1
-        Me.cmdLaunchGame.Text = frmLauncher_launchGameButton_text
+        Me.cmdLaunchGame.Size = New System.Drawing.Size(179, 37)
+        Me.cmdLaunchGame.TabIndex = 0
+        Me.cmdLaunchGame.Text = Global.Launcher.My.Resources.Resources.frmLauncher_launchGameButton_text
```
Comment (FWest98) [inline Launcher/Launcher/Launcher.Designer.vb:44]: Why did you change all the references to the resources?

edit: I guess VS does that automatically. But it's no problem, they still work.

File: Launcher/Launcher/Launcher.Designer.vb [hunk 2]
```diff
@@ -98,7 +98,7 @@ Partial Class frmLauncher
         Me.FormBorderStyle = System.Windows.Forms.FormBorderStyle.Fixed3D
         Me.MaximizeBox = False
         Me.Name = "frmLauncher"
-        Me.Text = frmLauncher_title
+        Me.Text = "OpenRCT2 Launcher"
```
Comment (FWest98) [inline Launcher/Launcher/Launcher.Designer.vb:101]: The titles should be edited in the resource files and not in the designer anymore.

**Comment by FWest98** [comment]
It builds, so I'd say it's OK. I'll fix the resource issues and change the login system a bit to a more modular approach. It prevents duplication of code everywhere.

**Comment by FWest98** [comment]
Oh, and @runelaenen, if you want to work on two pull requests at the same time, create different branches. That prevents commitcluttering like this. You could name them like 'feature-[name]'. That's following the conventions in this repo as well.

**Comment by runelaenen** [comment]
All right,
I've also forgot some things in the config.vb file (like setting the default values) since I only saw that I should've done that way later (and I edited a lot more in my project so I'm afraid I can't push it)

File: Launcher/Launcher/Launcher.Designer.vb (hunk 2 above)
Comment (runelaenen) [inline Launcher/Launcher/Launcher.Designer.vb:101]: Sorry, I didn't know that. i'll make sure to do this next time

Comment (FWest98) [inline Launcher/Launcher/Launcher.Designer.vb:101]: Great :)
